tests/at_end2end_test/lib/src/test_initializers.dart

```diff
@@ -16,17 +19,32 @@ class TestSuiteInitializer {
     return _singleton;
   }
 
-  Future<void> testInitializer(String atSign, String namespace) async {
+  Future<void> testInitializer(String atSign, String namespace,
+      {String authType = 'pkam'}) async {
```

Comment:
For clarity in the test cases which use this function, would it be better to make `authType` a required parameter so that it's always explicit in the test code which authType is being used?